blank-canvas-blocks/sass/blocks/_pullquote.scss

```diff
@@ -17,6 +17,7 @@
 			font-size: var(--wp--custom--pullquote--citation--typography--font-size);
 			font-family: var(--wp--custom--pullquote--citation--typography--font-family);
 			font-style: var(--wp--custom--pullquote--citation--typography--font-style);
+			font-weight: var(--wp--custom--pullquote--citation--typography--font-weight);
```

Comment:
Does this break for BCB and the other children that don't have this defined on their theme.json files? Maybe we should default it on BCB's theme.json to whatever the editor/browser has by default

Reply:
It appears to fail gracefully (the missing variable just seems to roll the property back to the browser-defined value of 400.

However it does seem dirty that BCB's CSS would use a variable that it doesn't express.  I suggest that we default that to 400 in BCB.

Additionally this scenario is EXACTLY what I'm excited for by styling elements of blocks in theme.json.  Once that expands beyond anchors and paragraphs we'll be able to remove that ponyfill from BCB entirely.